frontend/src/components/ScheduledEventsTable/ScheduledEventsTable.tsx

```diff
@@ -1,29 +1,39 @@
-import React, { FunctionComponent, ReactNode } from 'react';
+import classNames from 'classnames';
+import React, { FunctionComponent, ReactNode, useState } from 'react';
 import { Table } from 'react-bootstrap';
 import { ScheduledEvent } from '../../types/scheduledEvents';
 import { ScheduledEventsTableRow } from '../ScheduledEventsTableRow';
-
 export interface ScheduledEventTableProps {
   currentPage: number;
   pageLimit: number;
   events: Array<{}>;
+  handleRunHistory: (id: number, name: string) => void;
```

Comment:
Rename this to `onRowClick`, or `onRowSelect` ... it's not likely to be just about the run history ...may want to do something else when a user selects a row. Rename for all other occurrences, even for `ScheduledEventsTableCard`